Sources/Sentry/SentryStacktraceBuilder.m

```diff
@@ -25,6 +26,41 @@ - (id)initWithCrashStackEntryMapper:(SentryCrashStackEntryMapper *)crashStackEnt
     return self;
 }
 
+- (SentryStacktrace *)buildStacktraceForThread:(SentryCrashThread)thread
```

Comment:
`m`: There is some duplicate code in this function and `buildStacktraceForCurrentThread`. Can refactor this to reuse most of the code?